tests/Projection/PdoEventStoreReadModelProjectorTest.php

```diff
@@ -191,4 +191,39 @@ public function it_throws_exception_when_unknown_event_store_instance_passed():
             10
         );
     }
+
+    /**
+     * @test
+     */
+    public function it_dispatches_pcntl_signals_when_enabled(): void
```

Comment:
We should move these tests to the AbstractReadModelProjectorTest in the event-store repo. This way the test can be reused across all event-store implementations. If you think InMemoryEventStore is not worth adding this feature, we simply skip the tests for it.